modules/tax/src/Entity/TaxType.php

```diff
@@ -229,4 +236,18 @@ public function hasRate(TaxRateInterface $rate) {
     return array_search($rate, $this->rates) !== FALSE;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getGenericLabel() {
+    return $this->genericLabel;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setGenericLabel($genericLabel) {
+    $this->genericLabel = $genericLabel;
```

Comment:
You're missing the GenericLabel::assertExists($genericLabel); line in this method.